### PR TITLE
Remove form aliases

### PR DESCRIPTION
**Pull request by core23**
I am targeting this branch, because the 2.x branch looks dead.

## Changelog

```markdown
### Fixed
- Remove form aliases
```

## Subject

This will now work with symfony 3.

### CODE REVIEW

**Comment by greg0ire** [comment]
Please fix Travis

**Comment by core23** [comment]
Fixed

File: src/ProductBundle/Block/RecentProductsBlockService.php [hunk 1]
```diff
@@ -79,10 +82,14 @@ public function validateBlock(ErrorElement $errorElement, BlockInterface $block)
...
     {
-        $formMapper->add('settings', 'sonata_type_immutable_array', [
+        $formMapper->add('settings', ImmutableArrayType::class, [
             'keys' => [
-                ['number', 'integer', ['required' => true]],
-                ['title',  'text',    ['required' => false]],
+                ['number', IntegerType::class, [
+                    'required' => true,
+                ]],
+                ['title',  TextType::class,    [
```
[the remark this answers is not in the document]
Comment (core23) [reply src/ProductBundle/Block/RecentProductsBlockService.php]: Fixed

**Comment by greg0ire** [comment]
Thanks @core23 !

**Pull request merged**